tests/testthat/test-param-estimates.R

```diff
@@ -1,31 +1,59 @@
-context("Test bbi summary functions")
+context("Test param_estimates functions")
 
 if (Sys.getenv("METWORX_VERSION") == "" && Sys.getenv("DRONE") != "true") {
   skip("test-param-estimates only runs on Metworx or Drone")
 }
 
+setup({
```

Comment:
`testthat` 3e [will deprecate](https://testthat.r-lib.org/articles/third-edition.html#changes-1) `setup()` and `teardown()`, now may be the time to start thinking about another pattern, i.e., test fixtures.

Reply:
oof. This is used in a few places. I shall "start thinking" about it and we'll cross that bridge when we decide to switch. Functionally, this won't be hard to replicate with fixtures.